src/Microsoft.DotNet.Web.Spa.ProjectTemplates/ReactRedux-CSharp.csproj.in

```diff
@@ -46,12 +51,12 @@
 
     <!-- Include the newly-built files in the publish output -->
     <ItemGroup>
-      <DistFiles Include="wwwroot\dist\**" />
+      <DistFiles Include="wwwroot\dist\**; ClientApp\dist\**" />
```

Comment:
This is weird. GitHub is showing this file as an updated rename of `VueSpa.csproj`, but that project template shouldn't have this change in it. But then I see it ends up being `ReactReduxSpa.csproj`, which should have this line in it.

So I'm guessing that it's just Git/GitHub's rename detection logic going wrong, as opposed to you adding new stuff to the template configs. Please let me know if you have any reason to think otherwise!

Reply:
Yeah, the git rename detection is a little goofy here. It gets messed up because VueSpa, ReactReduxSpa, and ReactSpa are identical files